src/components/HeroSection/HeroSection.scss

```diff
@@ -3,87 +3,46 @@
 
 .hero {
   background: $bic_black;
-  display: flex;
-  position: relative;
+  background-image: url("../../images/crowd.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
 
-  &__bg {
-    overflow: hidden;
-    position: absolute;
-    width: 100%;
-    height: 100%;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    z-index: 0;
-
-    img {
-      height: 100%;
-      width: 100%;
-      object-fit: cover;
-    }
-  }
-
-  & .site-section {
-    z-index: 1;
-  }
+  // Creating rows in the explicit grid for the benefit of using the full possibilities of grid-row.
+  grid-template-rows: repeat(2, minmax(0, auto));
 }
 
-.content-grid {
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: min-content 1fr;
-  grid-template-areas:
-    "logo image"
-    "text image";
-  flex: 1;
-  gap: $spacing-x-small;
-  margin: $spacing-large 0;
-  z-index: 1;
+.nota-hero-logo {
```

Comment:
Updating classname because I notice the class was being overridden by another class named logo we use on the DUD portfolio version of the site.